bin/citgm.js

```diff
@@ -70,40 +75,42 @@ if (!citgm.windows) {
 
 app.parse(process.argv);
 
+var config = app.config ? util._extend(require(resolve(process.cwd(), app.config)), app) : app;
```

Comment:
I was suggesting that `require.resolve` could be used instead, but in retrospect this module is a bit cleaner.

We do have the policy for rampant module abuse for the sake of more integration testing so I'm ok with it !